### ISSUE TITLE
Choose between focus and active for buttons & links

### ISSUE THREAD

**Issue by foxwoods369** [zurb/foundation-sites#10352]
By default it seems that foundation uses ':focus' instead of ':active' for styling when a link or button is active (i.e. has just been clicked). This seems to mean that when a user hits back, if the last control they clicked was a link or button, it will continue to be highlighted as if it was just clicked (only tested on firefox & android). There doesn't seem to be a good way to configure this functionality to use ':active' instead of ':focus' to avoid this and properly reset the control back to an inactive state after going back in the browser. 

Can this be added in a future foundation release? If not, is there a recommended way to address this? My current thoughts were either forking or adding a post-process step to switch all :focus to :active, though that may have unintended effects.

**Comment by IamManchanda** [comment]
Yup true https://drive.google.com/file/d/0B4EN5uGMUjoZMUJaOGwwMWhrU0E/view

cc @kball @brettsmason

**Comment by foxwoods369** [comment]
Giving this a quick bump: any guidance on how to support this? I'm happy to submit a PR with the recommended change.

**Comment by DanielRuf** [comment]
Hm, but focus and active are two totally different states. Both can be also chained afaik.

**Comment by colin-marshall** [comment]
I agree @DanielRuf. Also this is similar to what's being discussed in #6105.

**Comment by foxwoods369** [comment]
Similar, but that seems to be without leaving the page. In my case I presume the browser is remembering the focus on the page one navigates back to.

**Comment by colin-marshall** [comment]
Is this happening for you in the latest versions of Firefox?
